Tchou-Tchou/StatusMenuController.swift

```diff
@@ -4,49 +4,67 @@ import CoreWLAN
 class StatusMenuController: NSObject {
     @IBOutlet var statusMenu: NSMenu!
     let statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.variableLength)
-
+    
```

Comment:
This is a general comment for all the similar things in this PR, but please avoid committing changes like spaces on empty lines :)